internal/runners/config/get.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"github.com/ActiveState/cli/internal/config"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/logging"
+	"github.com/ActiveState/cli/internal/output"
+)
+
+type Get struct {
+	out output.Outputer
+	cfg *config.Instance
+}
+
+type GetParams struct {
+	Key Key
+}
+
+func NewGet(prime primeable) *Get {
+	return &Get{prime.Output(), prime.Config()}
+}
+
+func (g *Get) Run(params GetParams) error {
+	value := g.cfg.Get(params.Key.String())
+	if value == nil {
+		return locale.NewInputError("err_config_not_found", "No config value for key: {{.V0}}", params.Key.String())
+	}
+
+	err := rules.Get(params.Key).getEvent()
```

Comment:
This doesn't appear to affect the `value`. The point of the event is to be able to interpret and modify the value.